src/app/components/top-sellers/top-sellers.component.html

```diff
@@ -0,0 +1,22 @@
+<h1 class="titleTopSellers">Más vendidos</h1>
+<div *ngIf="productSellers && productSellers.length > 0" class="carousel-container">
+  <button *ngIf="controls" class="btn-carousel" (click)="onPrevClick()">
+    <i class="fa-solid fa-chevron-left"></i>
+  </button>
+  <div class="globalContainer">
+    <div *ngFor="let prod of productSellers; let i=index" [ngClass]="{'image-active': selectedIndex === i}"
+      class="productTopSellers">
+      <h1> {{prod.name | titlecase}} </h1>
+      <img [src]="prod.image" alt="ProductName" class="productImage">
```

Comment:
The alt image shouldn't be repeated.
```suggestion
      <img [src]="prod.image" [alt]="prod.image" class="productImage">
```

Reply:
prod.name